server/api/playbooks.go

```diff
@@ -138,6 +138,20 @@ func (h *PlaybookHandler) createPlaybook(w http.ResponseWriter, r *http.Request)
 		}
 	}
 
+	if pbook.WebhookOnArchiveURL != "" {
```

Comment:
QA just found a bug with this on my own implementation of the webhook on creation, and it reproduces here: if the UI toggle is disabled but an invalid URL was entered, the playbook cannot be created nor saved, so we need to change this condition:
```suggestion
	if pbook.WebhookOnArchiveEnabled {
```